artifactory/commands/transferfiles/delayedartifactshandler.go

```diff
@@ -129,16 +129,20 @@ func consumeDelayFilesIfNoErrors(phase phaseBase, addedDelayFiles []string) erro
 	return nil
 }
 
-func countDelayFilesContent(filePaths []string) (int, error) {
+func countDelayFilesContent(filePaths []string) (int, int64, error) {
```

Comment:
Let's use named returned values, to make it easier to quickly understand what the returned values are.